src/components/Books.jsx

```diff
@@ -30,14 +30,15 @@ function Books({
             <div className="progress-bar">
               <CircularProgressbar value="75" text="75%" />
             </div>
-            <div>
+            <div className="progress-text">
               <h2>
                 75
                 %
               </h2>
               <p>Completed</p>
             </div>
           </div>
+          <div className="Line-2" />
           <div className="section3">
             <p>CURRENT CHAPTER</p>
             <h3>Chapter 17</h3>
```

Comment:
As part of the project requirements, all elements should be added (you can even use a placeholder if need be). In view of this your design is missing the category ( `Action` ) on top of the book title ( `The Hunger Game` ) as shown in the images below. Kindly add it so that it will match the actual design. :pray: 

### Your design 

![Screenshot from 2023-06-13 18-26-41](https://github.com/misalazeem/bookstore-react/assets/65233860/233d6b25-a23d-4516-a2b0-1f1980845ab4)


### Actual design 

![Screenshot from 2023-06-13 18-26-53](https://github.com/misalazeem/bookstore-react/assets/65233860/b28aace1-77af-4ba6-a294-0732a17ad832)